tests/unit/s2n_ecc_evp_test.c

```diff
@@ -17,13 +17,30 @@
 
 #include "api/s2n.h"
 
+#include "crypto/s2n_fips.h"
 #include "crypto/s2n_ecc_evp.h"
 #include "stuffer/s2n_stuffer.h"
 #include "testlib/s2n_testlib.h"
+#include "tls/s2n_connection.h"
+#include "tls/s2n_security_policies.h"
 #include "utils/s2n_mem.h"
 
 #define ECDHE_PARAMS_LEGACY_FORM 4
 
+/**
+ * Small helper function that builds a client connection w/ specified version
+ * @param version Requested version for a particular security policy
+ * @param connection Mock connection - will have its config set to the requested version
+ * @return success status
+ */
+static int get_test_s2n_connection_with_version(const char* version, struct s2n_connection* connection) {
+    POSIX_ENSURE_REF(connection);
+    const struct s2n_security_policy* sec_policy;
+    EXPECT_SUCCESS(s2n_find_security_policy_from_version(version, &sec_policy));
+    connection->config->security_policy = sec_policy;
+    return S2N_SUCCESS;
+}
```

Comment:
This is unnecessary. You can just call s2n_connection_set_cipher_preferences and s2n_connection_get_security_policy.

Reply:
Will change